installscripts/dockerfiles/jenkins-ce/configure-cli.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
If we absolutely have to configure Jenkins after the fact, we need a better solution than replicating all of the Chef cookbook steps in a new shell script.

Let's talk about this. If we really need to run these post-init steps for both the Jenkins Docker and the standalone Jenkins, let's move this particular chunk of functionality and scripts out of the Chef cookbook and have both scenarios invoke the same config script, and not just duplicate the same steps in a different location in the code.

That approach would honestly be better all around, as today we copy all of these `jenkins-cli.jar` scripts over to the target box and run them there for scenario 1, and we really don't need to.

Reply:
I had moved the jenkins cli script into terraform so that all scenarios can re use the script. This script will run the jenkins cli operations (credentials, jobs etc). Need to test the scenario1 in this case and then we can remove the conifigure jenkins cli related functionalities. So there will be only one script for all the scenarios.

Need to test it. Will update once it is done.

Reply:
Right, but can you actually move all the `/cookbooks/jenkins/files/*/*.sh` scripts that cover the shared Jenkins configuration steps _out_ of the Chef cookbook, and create a single entry point to invoke all of them?

That single entry point/script can be called in-place for all scenarios and the correlated steps can be removed from Chef entirely.

Reply:
Note that if we go with this I am OK with copying the scripts to the Jenkins instance for Scenario 1 with Terraform, like we do with the XMLs currently.